### ISSUE TITLE
web-polymer template doesn't work

### ISSUE THREAD

**Issue by gonzalopezzi** [google/stagehand#313]
A new **web-polymer** project doesn't run as expected: the project is generated with no errors, the dependencies get updated correctly via pub get, but when you run the project with pub serve in Dartium or Chrome, the page is blank. 
Inspecting the DOM you can see an empty `<main-app></main-app>` tag. So I think polymer is not being initialized properly. 

It only happens with version **1.0.2** of stagehand. If I activate the previous version (1.0.1) the generated project works. I have reproduced the error in two Windows machines (WIndows 7 64 bits and WIndows 10 64 bits). 

**Steps to reproduce the problem**: 
(Activate the new version of stagehand)
`pub global activate stagehand`
(Generate the new project)
```
mkdir new_project
cd new_project
stagehand web-polymer
```
(Update dependencies)
`pub get`
(Serve the project)
`pub serve`
(Run in dartium or chrome localhost:8080 and the page is blank). 

**Workaround:**
Install a previous version of stagehand
```
pub global deactivate stagehand
pub global activate stagehand 1.0.1
mkdir new_project_2
cd new_project_2
stagehand web-polymer
pub get
pub serve
```
(The sample code works!)

The same problem happens when I create the project from WebStorm. I suppose it is using the latest version of stagehand internally.

**Comment by kwalrath** [comment]
Thanks for reporting this. The only difference that seems like it could cause this change is the removal of library directives: 3aec99d37e029615e3f1a2becd7d1a48737ca897

Looking into this a bit more, the problem is the removal of the `library` directive from `lib/main_app.dart`. It changed from this:

```
@HtmlImport('main_app.html')
library polymer_app_101.lib.main_app;

import 'dart:html';
```

to this:

```
@HtmlImport('main_app.html')
import 'dart:html';
```

Adding back a `library` directive to `lib/main_app.dart` fixes the problem.

@jakemac53 does that sound right to you? Is there a better fix than adding back the `library` directive to that file?

**Comment by jakemac53** [comment]
Yep, that is correct. The initialize package probably should throw an error in this case so the error is at least more discoverable.

**Comment by kwalrath** [comment]
I've released 1.0.3, and web-polymer is working for me. Thanks again for reporting this issue, @gonzalopezzi! Your well-written bug report made it easy for me to reproduce and then fix the problem.

**Comment by gonzalopezzi** [comment]
Great! Thanks for the quick fix.
